docs/serializer-formatter.rst

```diff
@@ -194,6 +194,23 @@ In the following example, we read both the key and the value of the messages in
       "key1" \t {"f1": "value1"}
 
 
+In the following example, we read both the key and the value of the messages in JSON
```

Comment:
suggest revise as:

```
In the following example, the schema ID, key, and value of the message is printed in JSON. 
During registration, |sr| assigns an ID for new schemas that is greater than the IDs of the 
existing registered schemas. The IDs from different |sr| instances may be different.
```

Reply:
@joel-hamill minor nit: the schema IDs are not printed in JSON, but the message key and value are printed in JSON. The first sentence in your suggestion makes it seem like all of them are printed in JSON. How about the following?

```
The following example prints the key and value of the message in JSON and the schema IDs
for the key and value.
During registration, |sr| assigns an ID for new schemas that is greater than the IDs of the 
existing registered schemas. The IDs from different |sr| instances may be different.
```